tests/test_workdir.py

```diff
@@ -1,136 +1,46 @@
-    assert Path(MOCK_CWD, "runs", "fluxsite", "tasks", "site_bar_R1_S0").exists()
-    assert Path(MOCK_CWD, "runs", "fluxsite", "tasks", "site_bar_R1_S1").exists()
-
-    shutil.rmtree(MOCK_CWD / "runs")
-
-    # Success case: test non-verbose output
-    with contextlib.redirect_stdout(io.StringIO()) as buf:
-        setup_fluxsite_directory_tree(fluxsite_tasks=tasks, root_dir=MOCK_CWD)
-    assert buf.getvalue() == (
-        f"Creating runs/fluxsite/logs directory: {MOCK_CWD}/runs/fluxsite/logs\n"
-        f"Creating runs/fluxsite/outputs directory: {MOCK_CWD}/runs/fluxsite/outputs\n"
-        f"Creating runs/fluxsite/tasks directory: {MOCK_CWD}/runs/fluxsite/tasks\n"
-        f"Creating runs/fluxsite/analysis directory: {MOCK_CWD}/runs/fluxsite/analysis\n"
-        f"Creating runs/fluxsite/analysis/bitwise-comparisons directory: {MOCK_CWD}"
-        "/runs/fluxsite/analysis/bitwise-comparisons\n"
-        f"Creating task directories...\n"
-    )
+    # Success case: generate the full fluxsite directory structure
+    setup_fluxsite_directory_tree()
 
-    shutil.rmtree(MOCK_CWD / "runs")
+    for path in setup_mock_fluxsite_directory_list():
+        assert path.exists()
 
-    # Success case: test verbose output
-    with contextlib.redirect_stdout(io.StringIO()) as buf:
-        setup_fluxsite_directory_tree(
-            fluxsite_tasks=tasks, verbose=True, root_dir=MOCK_CWD
-        )
-    assert buf.getvalue() == (
-        f"Creating runs/fluxsite/logs directory: {MOCK_CWD}/runs/fluxsite/logs\n"
-        f"Creating runs/fluxsite/outputs directory: {MOCK_CWD}/runs/fluxsite/outputs\n"
-        f"Creating runs/fluxsite/tasks directory: {MOCK_CWD}/runs/fluxsite/tasks\n"
-        f"Creating runs/fluxsite/analysis directory: {MOCK_CWD}/runs/fluxsite/analysis\n"
-        f"Creating runs/fluxsite/analysis/bitwise-comparisons directory: {MOCK_CWD}"
-        "/runs/fluxsite/analysis/bitwise-comparisons\n"
-        f"Creating task directories...\n"
-        f"Creating runs/fluxsite/tasks/site_foo_R0_S0: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_foo_R0_S0\n"
-        f"Creating runs/fluxsite/tasks/site_foo_R0_S1: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_foo_R0_S1\n"
-        f"Creating runs/fluxsite/tasks/site_bar_R0_S0: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_bar_R0_S0\n"
-        f"Creating runs/fluxsite/tasks/site_bar_R0_S1: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_bar_R0_S1\n"
-        f"Creating runs/fluxsite/tasks/site_foo_R1_S0: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_foo_R1_S0\n"
-        f"Creating runs/fluxsite/tasks/site_foo_R1_S1: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_foo_R1_S1\n"
-        f"Creating runs/fluxsite/tasks/site_bar_R1_S0: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_bar_R1_S0\n"
-        f"Creating runs/fluxsite/tasks/site_bar_R1_S1: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_bar_R1_S1\n"
-    )
+    shutil.rmtree(Path("runs"))
```

Comment:
This line can probably removed since the mock CWD is cleaned up at the end of all the tests